__tests__/workers/cdc.ts

```diff
@@ -130,6 +134,21 @@ const defaultUser: ChangeObject<User> = {
   notificationEmail: true,
 };
 
+describe('content requested', () => {
```

Comment:
This will always be true because of the way this test is designed. I would rethink it. Not sure it's needed

Reply:
Perfect!